Source/Android/app/src/main/res/values/strings.xml

```diff
@@ -909,4 +909,23 @@ It can efficiently compress both junk data and encrypted Wii data.
     <string name="about_support"><a href="https://forums.dolphin-emu.org/">Support</a></string>
     <string name="about_copyright_warning">\u00A9 2003–2015+ Dolphin Team. \u201cGameCube\u201d and \u201cWii\u201d are trademarks of Nintendo. Dolphin is not affiliated with Nintendo in any way.</string>
 
+    <!-- Tools -->
+    <string name="tools_submenu">Tools</string>
```

Comment:
```suggestion
```

Reply:
Have resolved the above and removed a couple of extra unused strings